app/helpers/navigation_helper.rb

```diff
@@ -0,0 +1,18 @@
+module NavigationHelper
+  def navigation_links_internal
+    links = [
+      { text: "Manuals", href: manuals_path, active: request.path == manuals_path },
+    ]
+    if Time.zone.today < Date.new(2023, 11, 1)
+      links.push({ text: "What's new", href: whats_new_path, active: request.path == whats_new_path })
```

Comment:
```suggestion
      links << { text: "What's new", href: whats_new_path, active: request.path == whats_new_path })
```
nit in ruby it's more idiomatic to use the `<<` operator.